# Genomics & Bioinformatics (GEN711) Final Project
## Rhizosphere Microbiome Community Analysis

### Introduction
The rhizosphere is the soil that directly surrounds a plant's roots and that soil's constituents which influence the plant's growth (microbial, fungal, chemical). There are many reasons why one may want to study the microbial or fungal community present in a given plant's rhizosphere, as it can influence a number of attributes such as plant growth, disease resistance, etc. When attempting to extract, amplify, and sequence DNA from rhizosphere microbes, the 16S conserved region is a good target. When attempting to extract, amplify, and sequence DNA from rhizosphere fungi, the ITS conserved region is a good target. In this case, the sequences used were bacterial 16S sequences.

This data was generated by Haley Nolen, a PhD candidate working in Anissa Poleatewich's plant pathology lab at UNH. Her experiments focus on the rhizospheres of quinoa and wild relatives of quinoa. This analysis was performed on 32 samples, each of which came with forward and reverse fastq files, as well as a metadata file containing information on all 32 samples.
### Methods

* Entered the QIIME2 environment via RON

* Imported sequences into QIIME2 artifact

* Read data is already demultiplexed. Summarized into visualization file

* Observed resulting quality plots and determined where to cut reads

![Forward](https://user-images.githubusercontent.com/103780894/168383467-f247a7d3-a340-4e84-a87e-66c64d0034d7.PNG)

Forward read quality plot.

![Reverse](https://user-images.githubusercontent.com/103780894/168383529-deba0c4f-63ae-4d48-91e1-dc9089948b69.PNG)

Reverse read quality plot.

* Cut reads at 244 for reverse, no cut for forward (denoising with DADA2)

* Converted output files (dns.qza, table.qza, rep-seqs.qza) to qzv files

* Downloaded metadata from dns.qzv and added to gen711_metadata.csv

* Ran a BLAST with read data to establish taxonomy (Done by Haley)

* Made a barplot from taxonomy output file

* Established a phylogeny of samples

* Established core diversity metrics

### Findings

![Tax](https://user-images.githubusercontent.com/103780894/168380968-7656b587-2384-407f-b3af-c4ad6a5e7b85.PNG)

The above figure shows the taxonomy barplot. Some samples had a 0 feature count, those samples are the ones without bars. Across all samples, the microbial community is primarily composed of proteobacteria, acidobacteria, actinobacteria, and firmicutes. There is a great deal of diversity within most of the samples, and between most samples.

![Bray](https://user-images.githubusercontent.com/103780894/168382049-f66f6492-cc40-4812-8d65-3adc51620a47.PNG)

This figure is a Bray-Curtis plot, output from the core diversity metrics step. The samples, represented by dots, are colored by host plant, of which there were 3. In red is BVM, a wild relative of quinoa. In blue is a cross of BVM x quinoa. In orange is quinoa. The axes represent two different measures of diversity, which allows useful information to be determined by observing which samples cluster together. In this case, the samples are clearly not clustering by host, meaning that there is some deal of diversity between rhizosphere communities even within the same species of plant.

Although the expectation was to see some sort of clustering related to host species, the data is still useful. The taxonomy, especially, is important in that it shows the types of microbes that may have a large influence on plant growth, nutrient availability, and disease resistance.


